app/src/main/java/com/peacecorps/pcsa/StatusBarColorUtil.java

```diff
@@ -0,0 +1,30 @@
+package com.peacecorps.pcsa;
+
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.os.Build;
+import android.support.v4.content.ContextCompat;
+import android.view.Window;
+import android.view.WindowManager;
+
+/**
+ * Created by HIman$hu on 3/7/2017.
+ */
+
+public final class StatusBarColorUtil {
+
+    private static Context context;
+    private static Window window;
+
+    public StatusBarColorUtil(Context context, Window window) {
+        this.context = context;
+        this.window = window;
+    }
+
+    @TargetApi(Build.VERSION_CODES.LOLLIPOP)
+    public void changeStatusBarColor() {
```

Comment:
Make this method a public static one. This is kind of a Util class practice. You do not create an instance of Util classes. 
Ex: 
public static void changeStatusBarColor(Context context, Window window){
    if (VERSION CHECK) { // move version check to here since it is common to all activities
        //do status bar update
    }
}

Then you can simply call this method similar to below
StatusBarColorUtil.changeStatusBarColor(this,getWindow());